### PR TITLE
Add version info in heartbeat.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -51,6 +51,10 @@ include(GitInfoConfig)
 include(NebulaCustomTargets)
 include(ClangTidy)
 
+if (NEBULA_BUILD_VERSION)
+    add_compile_definitions(NEBULA_BUILD_VERSION=${NEBULA_BUILD_VERSION})
```
[the remark this answers is not in the document]
Comment (Shylock-Hg) [reply CMakeLists.txt:55]: See the conditional compile

File: src/common/version/Version.cpp.in [hunk 2]
```diff
@@ -14,6 +14,15 @@ std::string gitInfoSha() {
     return "@NEBULA_SERVER_GIT_INFO_SHA@";
 }
 
+std::string simpleVersionString() {
```
Comment (laura-ding) [inline src/common/version/Version.cpp.in:17]: you can use resue the `versionString()`, versionString(bool withLicenseInfo)

File: CMakeLists.txt (hunk 1 above)
Comment (laura-ding) [inline CMakeLists.txt:55]: add_definitions

**Review by laura-ding — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

File: src/common/clients/meta/MetaClient.cpp [hunk 3]
```diff
@@ -2361,6 +2362,9 @@ folly::Future<StatusOr<bool>> MetaClient::heartbeat() {
     req.set_host(options_.localHost_);
     req.set_role(options_.role_);
     req.set_git_info_sha(options_.gitInfoSHA_);
+#if defined(NEBULA_BUILD_VERSION)
+    req.set_version(simpleVersionString());
```
Comment (CPWstatic) [inline src/common/clients/meta/MetaClient.cpp:2366]: Could we set the version to `Not A Release` or something else if `NEBULA_BUILD_VERSION` was not set?

Comment (Shylock-Hg) [reply src/common/clients/meta/MetaClient.cpp:2366]: It's empty now. I think it's ok too.

**Pull request merged**